test/onboarding/onboarding.test.js

```diff
@@ -31,12 +32,14 @@ describe('Onboarding', () => {
                 headers: {},
                 body: JSON.stringify({}),
                 redirect: 'follow',
+                signal: new AbortController().signal,
```

Comment:
This isn't necessary, is it?
```suggestion
```

Reply:
Ah, I see that the signal is not actually a required property, but is just expected in the call to `jestFetch`. Nonetheless, this is a bit confusing, because you're passing that unused property to `sendRequest`. I think a bug could slip through there, because someone could modify `sendRequest` here:
https://github.com/mojaloop/finance-portal-lib/pull/32/files#diff-4da68f1a304cb0e37a29ce4ce3487349R20
to this:
```
        const result = await fetchFunc(url, { ...options });
```
and I think this test would still pass.

Reply:
Yes, I checked, if `sendRequest` is modified as I suggest, this test will still pass.